kw_webapp/models.py

```diff
@@ -271,15 +273,19 @@ class UserSpecific(models.Model):
     class Meta:
         unique_together = ("vocabulary", "user")
 
-    def answered_correctly(self, first_try=True):
```

Comment:
default value is used in only one test, so probably better to be explicit now that there are two flags